src/App.jsx

```diff
@@ -8,37 +10,37 @@ export const tabs = [
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
+function currentTab(id) {
+  const current = tabs.filter(el => el.id === id);
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+  if (current.length === 0) {
+    return tabs[0];
+  }
+
+  return current[0];
+}
+
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
+  const activeTab = currentTab(activeTabId);
+
+  return (
+    <div className="section">
+      <h1 className="title">Selected tab is {activeTab.title}</h1>
+
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <Tabs
+            tabs={tabs}
+            activeTabId={activeTab.id}
+            onTabSelected={idCurrent => setActiveTabId(idCurrent)}
```

Comment:
```suggestion
            onTabSelected={setActiveTabId}
```

Reply:
Here, we are using an arrow function (idCurrent => setActiveTabId(idCurrent)), which takes idCurrent as an argument and passes it to setActiveTabId. It’s an extra layer of indirection, where the function explicitly receives idCurrent and calls setActiveTabId(idCurrent).
Since setActiveTabId is already a function that takes idCurrent as an argument, this shorthand works the same way. Here, when onTabSelected is called with an argument, idCurrent will be passed directly to setActiveTabId.

So it's just a short version, and it will work the same way